lib/common/options.js

```diff
@@ -308,7 +308,12 @@ Options.defaults = {
     scrollBehavior: null,
     parseQuery: false,
     stringifyQuery: false,
-    fallback: false
+    fallback: false,
+    routeCreationOptions: {
```

Comment:
How about use `routesOptions` and make `strictRegexPaths` and `caseSensitivePaths ` be `pathToRegexpOptions` as same as route config .

```js
routesOptions: {
  //{strict: {true|false}, sensitive: {true|false}}
  pathToRegexpOptions: undefined,
  trailingSlash: false
}
```

Another important thing is that you should develop base on `dev` branch which options.defaults has been moved to `lib/common/options`.

Reply:
Sorry I push to 1.x because I think that dev branch it's the comming 2.x, I gona rebase the PR on dev branch with your recomendations